apps/bajour/components/bajour/best-of-wepublish.tsx

```diff
@@ -0,0 +1,264 @@
+    }
+  }
+`
+
+const BestOfWePublishFooter = styled('div')`
+  position: relative;
+  display: flex;
+  width: 100%;
+  background-color: #ffbaba;
+
+  > div {
+    background-color: white;
+    color: #ffbaba;
+    margin-left: 7%;
+    padding: 0 0.3rem;
+    display: inline-block;
+    height: 100%;
+  }
+
+  > span {
+    font-weight: 300;
+    padding: 0.5rem 0 2rem 1rem;
+    display: inline-block;
+    > b {
+      font-weight: 400;
+    }
+  }
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    > div {
+      padding: 0 1rem;
+    }
+    > span {
+      padding: 0.5rem 0 3rem 1.5rem;
+    }
+  }
+`
+
+const InnerContainer = styled('div')`
+  position: relative;
+  display: grid;
+  align-content: space-between;
+`
+
+const Content = styled('div')`
+  display: inline-block;
+  padding: 0.2rem;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+
+  > div {
+    // no other way of doing it and maintining current design i.e. line clamp and box would make the white bg 100% wide
+    max-height: 78px;
+    overflow: hidden;
+  }
+
+  span {
+    color: #2b2e34;
+    background: white;
+    padding: 1px 2rem 0.5px 0;
+  }
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    bottom: 0.8rem;
+    left: 0.8rem;
+
+    > div {
+      max-height: 82.5px;
+    }
+  }
+`
+
+const Title = styled('span')`
+  font-size: 14px;
+  font-weight: 600;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    font-size: 22px;
+  }
+`
+
+const Peer = styled('span')`
+  font-size: 12px;
+  font-weight: 600;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    font-size: 16px;
+  }
+`
+
+const MoreButton = styled('a')`
+  padding: 0.5rem 1.5rem;
+  background-color: white;
+  border: 3px solid #ffbaba;
+  color: #ffbaba;
+  border-radius: 0.5rem;
+  font-size: 20px;
+  position: absolute;
+  right: 2%;
+  bottom: -15%;
+  text-decoration: none;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    border-radius: 1rem;
+    padding: 1rem 2.5rem;
+    font-size: 27px;
+    right: 5%;
+    bottom: -30%;
+  }
+`
+
+const BestOfWePublish = ({numColumns, teasers, blockStyle, className}: BestOfWePublishProps) => {
+  return (
+    <BestOfWePublishWrapper>
+      <BestOfWePublishHeader>
+        <span>Best of We.Publish</span>
+      </BestOfWePublishHeader>
+      <Wrapper>
+        {teasers.map(teaser => {
+          const values = {
+            image: teaser?.image || teaser?.article?.image,
+            href: teaser?.article?.url,
+            title: teaser?.title || teaser?.article?.title,
+            peerName: teaser?.peer?.name,
+            peerLogo: teaser?.peer?.profile?.logo.squareSmallURL || ''
+          }
+
+          return (
+            <LinkWrapper color="inherit" underline="none" href={values.href} key={values.title}>
+              <PeerLogoWrapper>
+                {/* not possible to use next/image with dynamic domains */}
+                {/* eslint-disable-next-line @next/next/no-img-element */}
+                <img
```

Comment:
Why are we not using the Image component from website builder?

Reply:
for some reason nextjs or rather next/image requires to have a list of supported domains defined in next config. But in our case, it's not possible as the images defined by media can be on any domain so it's impossible to list them all

https://nextjs.org/docs/app/api-reference/components/image#remotepatterns